lib/api/sentry_api.dart

```diff
@@ -1,5 +1,13 @@
+import 'dart:convert';
 import 'dart:io';
+
 import 'package:http/http.dart';
+import 'package:async/async.dart';
```

Comment:
should this be `dart:async` instead?
```suggestion
import 'dart:async';
```

Reply:
It does not import/recognize the Result class when i use 'dart:async':

<img width="355" alt="Bildschirmfoto 2020-11-10 um 18 39 58" src="https://user-images.githubusercontent.com/3984453/98710638-395d4080-2384-11eb-9216-87ba745223e7.png">